packages/two_dimensional_scrollables/lib/src/table_view/table.dart

```diff
@@ -406,31 +448,70 @@ class RenderTableViewport extends RenderTwoDimensionalViewport {
     return false;
   }
 
-  // Updates the cached metrics for the table.
-  //
-  // Will iterate through all columns and rows to define the layout pattern of
-  // the cells of the table.
-  //
-  // TODO(Piinks): Add back infinite separately for easier review, https://github.com/flutter/flutter/issues/131226
-  // Only relevant when the number of rows and columns is finite
-  void _updateAllMetrics() {
-    assert(needsDelegateRebuild || didResize);
-
-    _firstNonPinnedColumn = null;
-    _lastNonPinnedColumn = null;
-    double startOfRegularColumn = 0;
-    double startOfPinnedColumn = 0;
+  // Updates the cached column metrics for the table.
```

Comment:
To help future readers: maybe document the meaning of the arguments to this method here since they are not immediately clear.

Reply:
Done!